lib/trie/trie.go

```diff
@@ -433,8 +433,6 @@ func (t *Trie) insertInBranch(parentBranch *node.Branch, key, value []byte) (new
 
 	oldParentIndex := parentBranch.Key[commonPrefixLength]
 	remainingOldParentKey := parentBranch.Key[commonPrefixLength+1:]
-
-	parentBranch.SetDirty(true)
```

Comment:
why was this removed?

Reply:
Sorry git rebase glitch, no longer removed.